sysvinit/openbsd/cloudinit.tmpl

```diff
@@ -0,0 +1,34 @@
+## template:jinja
+#!/bin/ksh
+
+# PROVIDE: cloudinit
+# REQUIRE: cloudinitlocal
+
+daemon="cloud-init"
+daemon_execdir="{{prefix}}/bin"
+daemon_flags="init"
+daemon_rtable=0
+daemon_timeout=1
+daemon_user=root
+
+. /etc/rc.d/rc.subr
+
+rc_bg="YES" # (undefined or "YES")
+rc_usercheck="YES" # (undefined or "NO")
+
+rc_start() {
+    test -e /etc/cloud/cloud-init.disabled \
```

Comment:
same as above

(but isn't it funny what reviewers can suddenly see when the files aren't full of comments ;)